src/ast-utils/declaration-property-handlers.ts

```diff
@@ -21,48 +24,47 @@ var analyzeProperties = require('./analyze-properties');
  * @return {object.<string,function>}      An object containing property
  *                                         handlers.
  */
-function declarationPropertyHandlers(declaration) {
+export function declarationPropertyHandlers(declaration:ElementDescriptor): PropertyHandlers {
   return {
-    is: function(node) {
+    is: function(node: estree.Node) {
       if (node.type == 'Literal') {
-        declaration.is = node.value;
+        declaration.is = (<estree.Literal>node).value.toString();
       }
     },
-    properties: function(node) {
-
+    properties: function(node: estree.Node) {
       var props = analyzeProperties(node);
 
       for (var i=0; i<props.length; i++) {
         declaration.properties.push(props[i]);
       }
     },
-    behaviors: function(node) {
+    behaviors: function(node: estree.Node) {
       if (node.type != 'ArrayExpression') {
         return;
       }
-
-      for (var i=0; i<node.elements.length; i++) {
-        var v = astValue.expressionToValue(node.elements[i]);
-        if (v === undefined)
+      const arrNode = <estree.ArrayExpression>node;
+      for (const element of arrNode.elements) {
+        let v = astValue.expressionToValue(element);
+        if (v === undefined) {
           v = astValue.CANT_CONVERT;
+        }
         declaration.behaviors.push(v);
       }
     },
-    observers: function(node) {
+    observers: function(node: estree.Node) {
       if (node.type != 'ArrayExpression') {
         return;
       }
-      for (var i=0; i<node.elements.length; i++) {
-        var v = astValue.expressionToValue(node.elements[i]);
+      const arrNode = <estree.ArrayExpression>node;
+      for (let element of arrNode.elements) {
```

Comment:
:+1: to migrating all of them eventually. I *think* there was a reason why I did it here? I can't seem to find one though..